test/add_constant_test.go

```diff
@@ -21,4 +21,13 @@ func TestAddConstantWithArguments(t *testing.T) {
 			"ignoreFuncs": "os\\.(CreateFile|WriteFile|Chmod|FindProcess),\\.Println,ignoredFunc,\\.Info",
 		}},
 	})
+	testRule(t, "add_constant", &rule.AddConstantRule{}, &lint.RuleConfig{
+		Arguments: []any{map[string]any{
+			"max-lit-count": "2",
+			"allow-strs":    "\"\"",
+			"allow-ints":    "0,1,2",
+			"allow-floats":  "0.0,1.0",
+			"ignore-funcs":  "os\\.(CreateFile|WriteFile|Chmod|FindProcess),\\.Println,ignoredFunc,\\.Info",
```

Comment:
This could be cleaner

```suggestion
			"allow-strs":    `""`,
			"allow-ints":    "0,1,2",
			"allow-floats":  "0.0,1.0",
			"ignore-funcs":  `os\.(CreateFile|WriteFile|Chmod|FindProcess),\.Println,ignoredFunc,\.Info`,
```